tools/plot/cli.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2025 Thousand Brains Project
+#
+# Copyright may exist in Contributors' modifications
+# and/or contributions to the work.
+#
+# Use of this source code is governed by the MIT
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/MIT.
+
+import argparse
+import logging
+import sys
+from pathlib import Path
+
+from tools.plot import objects_evidence_over_time, pose_error_over_time
+
+monty_root = Path(__file__).resolve().parent.parent.parent
+sys.path.append(str(monty_root))
+
+
+def main():
+    parent_parser = argparse.ArgumentParser(add_help=False)
+    parent_parser.add_argument(
+        "--debug",
+        action="store_true",
+        help="Enable debug logging",
+        dest="debug",
+    )
+
+    parser = argparse.ArgumentParser(
+        description="Plot a figure", parents=[parent_parser]
+    )
+    subparsers = parser.add_subparsers(dest="command")
+
+    objects_evidence_over_time.add_subparser(subparsers, parent_parser)
+    pose_error_over_time.add_subparser(subparsers, parent_parser)
```

Comment:
I'm confused. How does this work? Looking at [the `argparse` documentation](https://docs.python.org/3/library/argparse.html#sub-commands), I don't see anything about this feature.

It looks like you're calling `add_parser` on a function object, but where is that defined?

---

Never mind, I see where it's defined. That is very confusing because the module and the function in the module have the same name, and I didn't see that we were importing the module and not the function from the module.

I feel like this could be handled better to prevent confusion but I'm not sure the best way to do that. Maybe rename the functions to not match the module names?